modules/gapi/samples/semantic_segmentation.cpp

```diff
@@ -47,6 +47,52 @@ std::string get_weights_path(const std::string &model_path) {
     CV_Assert(ext == ".xml");
     return model_path.substr(0u, sz - EXT_LEN) + ".bin";
 }
+
+void classesToColors(const cv::Mat &out_blob,
+                           cv::Mat &maskImg) {
+    const int H = out_blob.size[0];
+    const int W = out_blob.size[1];
+
+    maskImg.create(H, W, CV_8UC3);
```

Comment:
@mpashchenkov I would rename to `mask_img` to keep consistency with naming

Reply:
Ok.
Do you refuse to use `cv::applyColorMap` instead?

Reply:
I just don't want to have a huge diff.